btcpay-down.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash
 
-. /etc/profile.d/btcpay-env.sh
+if [[ "$OSTYPE" == "darwin"* ]]; then
+	# Mac OS
+	BASH_PROFILE_SCRIPT="$HOME/btcpay-env.sh"
+
+else
+	# Linux
+	BASH_PROFILE_SCRIPT="/etc/profile.d/btcpay-env.sh"
+fi
+
+. ${BASH_PROFILE_SCRIPT}
```

Comment:
add `"`

Reply:
You mean `. ${BASH_PROFILE_SCRIPT}` should become `. "$BASH_PROFILE_SCRIPT"` ?
Is this causing issues for you? Syntax looks OK to me.

Reply:
@woutersamaey well with `. ${BASH_PROFILE_SCRIPT}` if there is a space in `$BASH_PROFILE_SCRIPT` this would fail right?

Reply:
Right. Figured this out on my own and already commited new changes.

Reply:
you did not fixed this one btw!

Reply:
Added the quotes now.